{{cookiecutter.project_shortname}}/scripts/bootstrap

```diff
@@ -25,7 +25,7 @@ if [ ${#flags[0]} -eq 0 ]; then
     flags=("--dev" "--skip-lock")
 fi
 
-pipenv install ${flags[@]}
+pipenv install ${flags[@]} --pre --dev --skip-lock
```

Comment:
`--dev` and `--skip-lock` are the default flags, if no flags are passed to the script, perhaps you could simply add `--pre` to the list `flags=("--dev" "--skip-lock" "--pre")`